ts/utils/hooks/bottomSheet.tsx

```diff
@@ -75,40 +75,48 @@ export const bottomSheetContent = (
   return {
     content: bottomSheetBody,
     config: {
-      snapPoints: [Math.min(snapPoint, Dimensions.get("window").height)],
       backdropComponent: () => BlurredBackgroundComponent(onClose),
       handleComponent: header
     }
   };
 };
 
+/**
+ * @typedef BottomSheetOptions
+ * @type {BottomSheetOptions}
+ * @property {component} component The React.Element to be rendered inside the bottom sheet body
+ * @property {title} title String or React.Element to be rendered as bottom-sheet header title
+ * @property {footer} footer React.Element or undefined to be rendered as sticky footer of our bottom sheet
+ * @property {snapPoint} snapPoint The array of points used to pin the height of the bottom sheet
+ * @property {onDismiss} onDismiss The possible function to be used as an effect of the dismissal of the bottom sheet
+ */
+type BottomSheetOptions = {
+  component: React.ReactNode;
+  title: string | React.ReactNode;
+  snapPoint: NonEmptyArray<number>;
```

Comment:
The base library supports bot number and string snap points (of type `Array<string | number>`). 
This allows us to specify snap points relative to the screen size (eg. `50%`). What do you think about enabling this feature also for our wrapper function?

Reply:
Yep! sure, [1fa5cd2](https://github.com/pagopa/io-app/pull/4631/commits/1fa5cd2a0d45bde0225816613cd144ba75bb3fc1) implements